llmware/library.py

```diff
@@ -517,7 +742,33 @@ def export_library_to_txt_file(self, output_fp=None, output_fn=None, include_tex
     def export_library_to_jsonl_file(self, output_fp, output_fn, include_text=True, include_tables=True,
                                      include_images=False, dict_keys=None):
 
-        """ Exports collection of text chunks to a jsonl file """
+        """ Exports collection of text chunks to a jsonl file 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Exports collection of text chunks to a jsonl file.
```